### ISSUE TITLE
Set max power when over CN470 to compensate for unoptimised RF front end for this frequency band

### ISSUE THREAD

**Issue by MedadRufus** [ImperialSpaceSociety/LoRaMac-node#206]
Restrict channels for CN470 to the 8 uplink channels/8 downlink channels used by both the Helium Network and the things networks.  Then set the max possible power for the radio, as the front end on the tracker(filters, antenna) are tuned for 868 Mhz, not 470 Mhz used in China. I am hoping by using a high tx power from the radio IC, some signal will make it past those filters/unoptimised antenna.

The max tx power(EIRP) in this band is 19.15dBm == 82.2 mW. Plenty of power to work with.
![image](https://user-images.githubusercontent.com/26815217/135340983-31d45010-71ec-4ebd-ab01-d9d3075ed369.png). Source: https://lora-alliance.org/wp-content/uploads/2020/11/lorawan_regional_parameters_v1.0.3reva_0.pdf

**Comment by MedadRufus** [comment]
Set the max eirp higher than 19.15 dbm, as it will not exceed this due to the poor antenna/filter configuration.

**Comment by MedadRufus** [comment]
Do we have to also set the Downlink channel mask? we don't do it on US/AU. But may be needed for ABP, which won't know which channel the downlink will be received on.

**Comment by MedadRufus** [comment]
work in progress in #208

**Comment by MedadRufus** [comment]
Frequency band is TTN Community Network frequency plan for China, using sub-band 11

**Comment by MedadRufus** [comment]
consider making the packet size smaller, so that it has a lower chance of browning out mid tx. Probably will not help because there is no capacitance in the system(either in the solar panels or else where).

**Comment by MedadRufus** [comment]
Consider setting the data rate to dr_0 to maximise chance of reception

**Comment by MedadRufus** [comment]
Consider looking at ICSPACE22 packet size, when it was transmitting at 30 dbm(1000 mW) over the US. It could transmit once in a while.

**Comment by MedadRufus** [comment]
Code from ICSPACE22 says that the packet size of 26 bytes:
https://github.com/ImperialSpaceSociety/picotracker-Lora/blob/a7c9ff234aceb6246083b4a6d6d09181d32f317c/Software/Projects/B-L072Z-LRWAN1/Applications/LoRa/End_Node/LoRaWAN/App/src/main.c#L403-L468

```c
	
	uint32_t i = 0;
  
  AppData.Port = LPP_APP_PORT;
  
  AppData.Buff[i++] = cchannel++;
  AppData.Buff[i++] = LPP_DATATYPE_BAROMETER;
  AppData.Buff[i++] = ( cayenne_pressure >> 8 ) & 0xFF;
  AppData.Buff[i++] = cayenne_pressure & 0xFF;
  AppData.Buff[i++] = cchannel++;
  AppData.Buff[i++] = LPP_DATATYPE_TEMPERATURE; 
  AppData.Buff[i++] = ( cayenne_temperature >> 8 ) & 0xFF;
  AppData.Buff[i++] = cayenne_temperature & 0xFF;
	
	// TO be uncommented if humidity value is recorded
//  AppData.Buff[i++] = cchannel++;
//  AppData.Buff[i++] = LPP_DATATYPE_HUMIDITY;
//  AppData.Buff[i++] = cayenne_humidity & 0xFF;


  /* The maximum payload size does not allow to send more data for lowest DRs.
	 * Problem solved by setting the min data rates for the two regions(AU915 and US915) greater
	 * than DR3
	 */


	/* Using cayenne reference 
	 * https://github.com/MicrochipTech/Location-Tracking-using-SAMR34-and-UBLOX-GPS-Module/blob/master/src/CayenneLPP/lpp.c  
	 */
	AppData.Buff[i++] = cchannel++;
  AppData.Buff[i++] = LPP_DATATYPE_GPSLOCATION;
  AppData.Buff[i++] = cayenne_latitude >> 16;
  AppData.Buff[i++] = cayenne_latitude >> 8;
  AppData.Buff[i++] = cayenne_latitude;
	
  AppData.Buff[i++] = cayenne_longitude >> 16;
  AppData.Buff[i++] = cayenne_longitude >> 8;
  AppData.Buff[i++] = cayenne_longitude;
	
	
  AppData.Buff[i++] = ( cayenne_altitudeGps >> 16 ) & 0xFF; 
  AppData.Buff[i++] = ( cayenne_altitudeGps >> 8 ) & 0xFF;
  AppData.Buff[i++] = cayenne_altitudeGps & 0xFF;
		
		
	AppData.Buff[i++] = cchannel++;
  AppData.Buff[i++] = LPP_DATATYPE_ANALOG_INPUT;
  AppData.Buff[i++] = cayenne_load_voltage;
  AppData.Buff[i++] = cayenne_no_load_voltage;
  
  AppData.Buff[i++] = cchannel++;
  AppData.Buff[i++] = LPP_DATATYPE_DIGITAL_OUTPUT; 
  AppData.Buff[i++] = cayenne_GPS_sats;
    




  AppData.BuffSize = i;
	
	/* Based on the code on the STRKT01 tracker, the max bytes we can send
	 * is 30 bytes. It has to be at a datarate above DR_3. By default, the
	 * picotracker transmits at DR_4. The packet size is 26 bytes in the
	 * picotracker.
	 */
	
	#if RADIO_ENABLED
	LORA_send( &AppData, LORAWAN_DEFAULT_CONFIRM_MSG_STATE);
	#endif
  
```

**Comment by MedadRufus** [comment]
#271 confirms the correct channel list, but the channel mask has not been merged into the `icspace-dev` branch

**Comment by MedadRufus** [comment]
fixed in #303
